src/evict.c

```diff
@@ -708,3 +708,16 @@ int performEvictions(void) {
     return result;
 }
 
+int performEvictions(void) {
+    int result = doPerformEvictions();
```

Comment:
i don't think we should wrap performEvictions. there are already many things this function does that may justify various wrappers (like latency monitor samples), it would become complex if each of them would be in a wrapper.
instead, i think you can change the two places at the top that `return EVICT_OK` to a `goto evict_ok`, which will be at the bottom of the function, and then handle the metrics in the goto labels at the bottom (all are run before returning).

Reply:
I'm confused about `EVICT_FAIL`. `EVICT_FAIL` also means we are over the memory limit. So maybe this time should also be included? But time event will not run, just command event.

Reply:
yea, so let's also use `goto` for that one.
for some reason, i think this would be better than a wrapper function.
maybe i'm wrong.. let's ask @guybe7 for his opinion.

Reply:
The problem of `EVICT_FAIL` is `performEvictions` only is triggered by command. If there is no command, this time can't be updated. But this is rarely, so maybe we can ignore it.

Reply:
i think we're ok to ignore it.
the `current_eviction_exceeded_time` metric will still work fine.
but also, these metrics are about "eviction" not about "no-eviction".
unless we wanna rename them to "oom-...."

Reply:
i also feel we should try to solve all the stats updates inside `performEvictions` rather than wrap it